### PR TITLE
describe cross-device communication, with cited silverpush example

### PR DESCRIPTION
**Pull request by npdoty**
and connected to the threat of cross-device/cross-partition recognition

addresses #71

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/186.html" title="Last updated on Oct 25, 2022, 10:35 PM UTC (199895f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/186/1e6015c...199895f.html" title="Last updated on Oct 25, 2022, 10:35 PM UTC (199895f)">Diff</a>

### CODE REVIEW

**Review by jyasskin — APPROVED** [review]

**Review by samuelweiler — APPROVED** [review]

**Comment by npdoty** [comment]
Re-writes to Identity on the Web section have made this difficult to merge. Cross-device communication and the new text here should be included in describing a common threat to track users across partitions.

File: index.html [hunk 1]
```diff
@@ -1043,8 +1049,8 @@
 href="#principle-prevent-cross-partition-recognition">principle</a> here instead requires separation
 at [=partition=] boundaries.
 
-Cross-[=partition=] recognition is generally accomplished by either "supercookies" or <a>browser
-fingerprinting</a>.
+Cross-[=partition=] recognition is often accomplished by: "supercookies", <a>browser
+fingerprinting</a>, or <a>cross-device communication</a>.
```
Comment (darobin) [inline index.html:1053]:
```suggestion
fingerprinting</a>, or [=cross-device communication=].
```

File: index.html [hunk 2]
```diff
@@ -1066,6 +1072,12 @@
 
 See [[fingerprinting-guidance]] for how to mitigate this threat.
 
+<dfn>Cross-device communication</dfn> is communication between one device (or a site running
+client-side code on that device) and a site running on another device. For example, sounds or light
```
Comment (darobin) [inline index.html:1076]: I think that using "site" makes this a bit clunky. Maybe:

```
<dfn>Cross-device communication</dfn> is communication between code on one device and code running on another device.
```

**Review by darobin — CHANGES_REQUESTED** [review]
Small changes, I think this is a good addition.

**Comment by npdoty** [comment]
This now conflicts with the reorganized Identity section. Integrating this text will be easier to do once Identity is organized again, so that it's clear where to insert the technique of cross-device communication as a threat to accomplish cross-partition recognition.

**Pull request closed without merging**